GoogleForCreators/web-stories-wp
# Dashboard: Empty State Redesign

## Context

Updates the view when there are no stories to see based on: https://www.figma.com/file/NHHMwIfxMnz39NxqkrFb7R/Stories-Post-Stable?node-id=6464%3A521842
(from figma Mar 4, 2021 9:20 am MST)
<img width="566" alt="Screen Shot 2021-03-04 at 9 19 36 AM" src="https://user-images.githubusercontent.com/10720454/109994664-c2e5a500-7cca-11eb-90d2-325245cad3bc.png">

## Summary

Updates dashboard empty state. 

## Relevant Technical Choices

- Adds color and bw "desert" svgs. bw will be needed for empty libraries on the editor (I can't find a ticket for this and I know it's really low priority). 
- Subs out old empty content message for new structure. Keeping children structure so that we can have buttons when we ant without giving more opinions to the container. 

## To-do

- Took some liberties matching the styles for search results of 0, questions out to design via slack and am still waiting on feedback. For now I think consistency here is better and less jarring. 

## User-facing changes

**Old (Staging/Main)**
<img width="855" alt="Screen Shot 2021-03-04 at 9 24 27 AM" src="https://user-images.githubusercontent.com/10720454/109995390-7a7ab700-7ccb-11eb-824b-6e5d0db28437.png">

**New (this branch)**

![storybook views](https://user-images.githubusercontent.com/10720454/109997162-283a9580-7ccd-11eb-8ad2-f98ccc89853a.gif)

![Screen Shot 2021-03-04 at 8 25 21 AM](https://user-images.githubusercontent.com/10720454/109995472-8e261d80-7ccb-11eb-9fdd-62319eee23e9.png)
![Screen Shot 2021-03-04 at 9 34 10 AM](https://user-images.githubusercontent.com/10720454/109996817-d1cd5700-7ccc-11eb-98f9-1a215d5fd921.png)


## Testing Instructions

- You can see the empty states all updated in storybook under Dashboard/views/{page}/content 
- Log in as an author level user with no stories in an existing account that has stories (our local environments have this built in, user: `author`, password is same as admin unless you changed it) to see the no stories/blank canvas state without wiping your local environment. Discussion with Pascal here #6613 makes me think this is a bug but for now it's really helpful for testing this!) 
- To check out search empty messages just search for something in 'my stories' that has no results. 
- To see empty templates, be really fast on the first load of the view, this is mostly just a future proof for when templates come from an API. 
- Empty stories should have a button that says "explore templates", clicking this should take users to the "explore templates" view in dashboard. 


### QA

See above.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?

No

### Does this PR change what data or activity we track or use?

No

### Does this PR have a legal-related impact?

No

## Checklist




---



Fixes #6437
